1.1/spec/18-Annex-C.adoc

```diff
@@ -1,6 +1,149 @@
+They may include one or more http://www.opengis.net/ont/geosparql#hasGeometry[geo:hasGeometry] relations and further properties related to the feature and may be grouped into http://www.opengis.net/ont/geosparql#FeatureCollection[geo:FeatureCollection] instances where `geo:FeatureCollection` is a new class in GeoSPARQL 1.1 specifically for the description of collections of `geo:Feature` instances.
+
+The following example SPARQL query retrieves all feature collections of a given SPARQL endpoint. A third-party webservice may convert the query results to one or many geoformat serializations.  
+
+```turtle
+SELECT ?fcollection ?item ?rel ?val ?geom 
+WHERE {
+  ?fcollection rdf:type geo:FeatureCollection .
+  ?item rdfs:subClassOf* geo:Feature .
+  ?item dcterms:isPartOf ?fcollection . 
+  ?item ?rel ?val .
+  OPTIONAL {?item geo:hasGeometry/geo:hasSerialization ?geom}
+}
+```
+
+Similarly, for SQL backends of OGC webservices, geospatial _features_ are made explicit by defining an SQL query retrieving a set of triples which may be converted to a geodata format.
+
+The following example SPARQL query retrieves a collection of Feature instances (each `item`) and, optionally, their serialized geometries (`geom`):
+
+```turtle
+SELECT ?item ?rel ?val ?geom 
+WHERE {
+  ?item rdfs:subClassOf* geo:Feature .
+  ?item ?rel ?val .
+  OPTIONAL {?item geo:hasGeometry/geo:hasSerialization ?geom}
+}
+```
+
+This SPARQL query retrieves all features' properties and their values directly connected to an instance of all classes which are subclasses of http://www.opengis.net/ont/geosparql#Feature[geo:Feature].
+The query result is a list of triples which may be converted to a `geo:FeatureCollection` instance.
+
+`geo:FeatureCollection` match the OGC API Features's <<OGCAPIF>> concept of a FeatureCollection and thus may be exposed using an OGC API Features webservice.
+
+However, it needs to be remarked that this may not deliver the expected result in all cases. There are several reasons for this:
+
+- More information for the `geo:FeatureCollection` instance may be needed, in particular, an identifier and perhaps a label for it. If the back-end data store also contains information for the `geo:FeatureCollection` instance then this may be queried for to solve this issue. If not, the API might need to create such information
+- Maybe only information in a certain namespace is of interest. The solution is to apply FILTER expressions to the SPARQL query
+- The above query only queries directly connected properties and their values. Often geodata formats like GML are hierarchical and therefore need further resolution of property values depicted as URIs. If that is the case the query needs to be adjusted.
+
+==== C.2 Mappings from CQL statements to GeoSPARQL queries
+
+This section presents lists of equivalences between CQL statements and GeoSPARQL statements.
```

Comment:
This seems like a good place to explain what CQL means and to refer to its specification document